superface/super.json

```diff
@@ -108,6 +108,24 @@
         }
       }
     },
+    "storefront/product": {
+      "file": "./grid/storefront/product/profile.supr",
```

Comment:
I know it is caused, because this PR was opened already long ago. But the paths should point to files in `capabilities` and `providers` folders.